site/book/03-packages/02-exploring-a-package.md

```diff
@@ -0,0 +1,35 @@
+After you fetch a package to your local filesystem, you typically want to explore the package to
+understand how it is composed and how it can be customized for your needs. Given a kpt package is
+just an ordinary directory of human-readable YAML files, you can naturally use your favorite file
+explorer, shell commands, or editor to explore the package.
+
+kpt also provides the `tree` command which is handy for quickly viewing package hierarchy and the
+constituent packages, files, and resources:
+
+```shell
+$ kpt pkg tree wordpress/
+PKG: wordpress
+├── [Kptfile]  Kptfile wordpress
+├── [service.yaml]  Service wordpress
+├── deployment
+│   ├── [deployment.yaml]  Deployment wordpress
+│   └── [volume.yaml]  PersistentVolumeClaim wp-pv-claim
+└── PKG: mysql
```

Comment:
Not directly related to this, but it would be useful if the `tree` command showed whether a subpackage is dependent or independent.